utils.css

```diff
@@ -2,19 +2,19 @@
 :root{
     --primary-clr:#05386b;
     --secondary-clr:#04478a;
-    --danger-clr: #FF0000;
-    --warning-clr:rgb(234 233 12);
-    --success-clr:#0ebf28;
+    --danger-clr: #ed3535;
+    --warning-clr:#c1c1c0;
+    --success-clr:#26a338;
```

Comment:
Consistent naming like either --color-colorname or --colorname-color throughout will make it easier for you and everyone else as well.

Reply:
Yes, that I thought, but its just personal preference to use `clr` instead of `color`. I hope it justifies.